.github/workflows/build_and_release.yml

```diff
@@ -73,216 +72,17 @@ jobs:
           draft: true
           prerelease: false
 
-  # Create the Linux Packages
-  # This is done in containers provided by
-  # https://github.com/performous/performous-docker since
-  # github actions provides only Ubuntu
+  # Pull in the Linux build workflow
   Linux_Packages:
-    # The upload step needs the release to be created first so it
-    # can access the upload_url output. This step also needs the output from
-    # the version dteremination job to name things correctly.
-    name: Create Linux packages
+    name: Build the Linux packages
+    uses: ./.github/workflows/linux.yml
```

Comment:
Can we also do this for windows/osx? this greatly reduces the content for the actual pipeline

Reply:
Yeah, my thought was to do just Linux in this PR and then do the rest in another PR to make my changes more visible. I think doing all at once in this PR would overwhelm those reviewing that don't have as in-depth knowledge about the pipeline that we do. Also, I know that @Lord-Kamina is making a new script/method for the MacOS builds, so I wanted to give him the opportunity to make that change :-)